integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/AnsibleSteps.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.service.smartevents.integration.tests.steps;
+
+import java.time.Duration;
+
+import org.assertj.core.api.Assertions;
+import org.awaitility.Awaitility;
+
+import com.redhat.service.smartevents.integration.tests.context.TestContext;
+import com.redhat.service.smartevents.integration.tests.context.resolver.ContextResolver;
+import com.redhat.service.smartevents.integration.tests.resources.AnsibleResource;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+
+public class AnsibleSteps {
+
+    private TestContext context;
+
+    public AnsibleSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @And("^Job template (.*) exists$")
+    public void checkAnsibleJobTemplateExists(String jobTemplateName) {
+        Assertions.assertThat(AnsibleResource.jobTemplatesList(jobTemplateName)
+                .stream().anyMatch(s -> s.contains(jobTemplateName)))
+                .isTrue();
```

Comment:
Please use Assertj  native list assertion like:
`assertThat(AnsibleResource.jobTemplatesList(jobTemplateName).anyMatch(....);`
That way we get more contextual info in case of failure.